runtime.ts

```diff
@@ -150,7 +150,9 @@ export function resolveModule(
   try {
     fetchResponse = os.codeFetch(moduleSpecifier, containingFile);
   } catch (e) {
-    // TODO Only catch "no such file or directory" errors. Need error codes.
+    if (e.message === "file does not exist") {
```

Comment:
Bonus points if you can figure out how to add error code in this PR. If you want to leave that for later, I'm ok with that - but the TODO must remain. Something like this
```
// TODO Testing the exact format of e.message is very brittle - the error object should have error codes.
```